langchain4j-agentic/src/main/java/dev/langchain4j/agentic/internal/AgenticParameterNameResolver.java

```diff
@@ -0,0 +1,26 @@
+package dev.langchain4j.agentic.internal;
+
+import java.lang.reflect.Parameter;
+import dev.langchain4j.agentic.declarative.K;
+import dev.langchain4j.service.ParameterNameResolver;
+import dev.langchain4j.service.V;
+
+import static dev.langchain4j.agentic.internal.AgentUtil.stateName;
+
+public class AgenticParameterNameResolver implements ParameterNameResolver {
```

Comment:
```suggestion
public class DefaultParameterNameResolver implements AgenticParameterNameResolver {
```
We usually call the default implementation `Default*`

Reply:
This is not the default implementation, is the one specific of the agentic module. The default one is in an inner class inside the `ParameterNameResolver` interface.

Reply:
Oh sorry, my bad